src/include/pbs_share.h

```diff
@@ -44,7 +44,9 @@
 
 /* Formula special case constants */
 
-#define FORMULA_FSPERC 		"fair_share_perc"
+#define FORMULA_FSPERC 		"fairshare_perc"
+#define FORMULA_FSPERC_DEP	"fair_share_perc"
```

Comment:
This isn't used in parse.c.  It's a formula keyword.  Since the formula is opaque to us, so there is no real way to tell if they're using the deprecated keyword or not.